logback/src/test/java/com/linecorp/armeria/common/logback/RequestContextExportingAppenderTest.java

```diff
@@ -116,12 +116,6 @@ void tearDown() {
             count++;
             switch (level) {
                 case Status.WARN:
-                    if (s.getThrowable() != null) {
-                        logger.warn(s.getMessage(), s.getThrowable());
-                    } else {
-                        logger.warn(s.getMessage());
-                    }
-                    break;
```

Comment:
Revert?

Reply:
or did you just want to fall through because it's identical in ERROR?

Reply:
Yes, I've just picked up IntelliJ's suggestion. Identical code for both `WARN` and `ERROR`.

Reply:
I will revert it if I should keep strictly to the scope of the PR.

Reply:
No worries. One less broken window = Awesomeness :smile: